addons/phone_validation/models/res_partner.py

```diff
@@ -1,13 +1,15 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+from odoo import api, models, fields
 from odoo import api, models
 from odoo.addons.phone_validation.tools import phone_validation
 
 
 class Partner(models.Model):
     _name = 'res.partner'
     _inherit = ['res.partner']
+    phone_mobile_search = fields.Char("Phone/Mobile", store=False, search='_search_phone_mobile_search')
```

Comment:
Well actually res.partner inherit from mail.thread.phone in SMS app, not here. I think we could move the view part in SMS and do nothing for partner in phone_validation. As SMS is auto install it is transparent for users.

Reply:
done.